### PR TITLE
Add option to disable magic comment completions

### PR DESCRIPTION
**Pull request by hibiyasleep**

### CODE REVIEW

**Comment by TheRealSyler** [comment]
@hibiyasleep thanks for creating the pr, there are two small issue with the code (see code review), i will merge the pr once they are fixed.

**Comment by hibiyasleep** [comment]
Thank you, but I can't see any comments, maybe they're under 'Pending' state?

**Review by TheRealSyler — CHANGES_REQUESTED** [review]

**Comment by TheRealSyler** [comment]
@hibiyasleep ups, i forgot to request the changes, should be visible now.

File: src/autocomplete/autocomplete.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ class SassCompletion implements CompletionItemProvider {
 
     const config = workspace.getConfiguration();
     const disableUnitCompletion: boolean = config.get('sass.disableUnitCompletion');
+    const disableCommentCompletion: boolean = config.get('sass.disableUnitCompletion');
```
[the remark this answers is not in the document]
Comment (hibiyasleep) [reply src/autocomplete/autocomplete.ts]: oops, fixed!

**Pull request merged**